2-mandatory/2-financial-times.js

```diff
@@ -5,37 +5,85 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    let titleCharacters = [];
+    
+    for (let title of allArticleTitles) {
+    
+            if (title.length<=65) {
+             titleCharacters.push(title);
+        
+        }
+
+    }
+    return titleCharacters;
 }
 
+    
+
+
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
```

Comment:
This looks almost perfect to me! A couple of very minor points:
- Are you happy with the variable name `shortHeadlines`? Maybe think about what value this variable is holding - can you think of a better name for it?
- It's a good idea to keep an eye on indentation and spacing - this will make it easier for other developers to read your code. Can you see any indentation in this function that could be improved? 😄